snippets/price.liquid

```diff
@@ -34,49 +34,41 @@
         - div.price__regular: Displayed when there are no variants on sale
         - div.price__sale: Displayed when a variant is a sale
-        - div.price__availability: Displayed when the product is sold out
     {%- endcomment -%}
     <div class="price__regular">
-      <dt>
-        <span class="visually-hidden visually-hidden--inline">{{ 'products.product.price.regular_price' | t }}</span>
-      </dt>
-      <dd {% if show_badges == false %}class="price__last"{% endif %}>
-        <span class="price-item price-item--regular">
-          {{ money_price }}
-        </span>
-      </dd>
+      <span class="visually-hidden visually-hidden--inline">{{ 'products.product.price.regular_price' | t }}</span>
+      <span class="price-item price-item--regular">
+        {{ money_price }}
+      </span>
     </div>
     <div class="price__sale">
-      <dt class="price__compare">
+      {%- unless product.price_varies == false and product.compare_at_price_varies %}
         <span class="visually-hidden visually-hidden--inline">{{ 'products.product.price.regular_price' | t }}</span>
-      </dt>
-      <dd class="price__compare">
-        <s class="price-item price-item--regular">
-          {% if settings.currency_code_enabled %}
-            {{ compare_at_price | money_with_currency }}
-          {% else %}
-            {{ compare_at_price | money }}
-          {% endif %}
-        </s>
-      </dd>
-      <dt>
-        <span class="visually-hidden visually-hidden--inline">{{ 'products.product.price.sale_price' | t }}</span>
-      </dt>
-      <dd {% if show_badges == false %}class="price__last"{% endif %}>
-        <span class="price-item price-item--sale">
-          {{ money_price }}
+        <span>
+          <s class="price-item price-item--regular">
+            {% if settings.currency_code_enabled %}
+              {{ compare_at_price | money_with_currency }}
+            {% else %}
+              {{ compare_at_price | money }}
+            {% endif %}
+          </s>
         </span>
-      </dd>
+      {%- endunless -%}
+      <span class="visually-hidden visually-hidden--inline">{{ 'products.product.price.sale_price' | t }}</span>
+      <span class="price-item price-item--sale price-item--last">
+        {{ money_price }}
+      </span>
     </div>
     <small class="unit-price caption{% if product.selected_or_first_available_variant.unit_price_measurement == nil %} hidden{% endif %}">
-      <dt class="visually-hidden">{{ 'products.product.price.unit_price' | t }}</dt>
-      <dd {% if show_badges == false %}class="price__last"{% endif %}>
+      <span class="visually-hidden">{{ 'products.product.price.unit_price' | t }}</span>
+      <div class="price-item price-item--last">
         <span>{{- product.selected_or_first_available_variant.unit_price | money -}}</span>
         <span aria-hidden="true">/</span>
         <span class="visually-hidden">&nbsp;{{ 'accessibility.unit_price_separator' | t }}&nbsp;</span>
```

Comment:
Could we adjust this string? The `uppercase` may render "P-E-R" when announced by screen readers. Removing the `uppercase` announces as expected but still may not be clear. Can we consider "for every" or "for each"? 
/cc @gretchen-willenberg

Reply:
@svinkle  cc'ing @melissaperreault to consult on the user of uppercase design. From a text perspective, "per" is used as as part of a product resource `Cost per item` label (attached). In Dawn, the only setting use was the Collection product grid setting (attached). 

 I'm fine with `Products on a page` for Collection setting, but the string above seems to be pricing. We should request updating it in the product resources as well. `Unit price` would be shorter and is used (although a bit B2B)

![Screen Shot 2021-09-28 at 3 18 44 PM](https://user-images.githubusercontent.com/68568732/135161869-25921233-ea56-436c-b547-9ffc0de947ec.png)


![Screen Shot 2021-09-28 at 3 19 31 PM](https://user-images.githubusercontent.com/68568732/135161885-14508e8c-4adf-4d9e-9f56-bc2838b05235.png)

Reply:
This bit of content is visually hidden so there'd be no visual change, only how the content is announced.

Let's keep the content as-is for now, perhaps address in another PR. I'm keen to have usability testing done with Fable soon. If this comes up as an issue we can revisit then.